### PR TITLE
feat(organization): user resource and data source

### PR DESCRIPTION
**Pull request by Serpentiel**
## About this change—what it does

adds organization user resource and data source

## Why this way

to have them in place

### CODE REVIEW

**Review by byashimov — CHANGES_REQUESTED** [review]

File: internal/sdkprovider/service/organization/organization_user.go [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+eliminate the member from the organization if one has accepted an invitation previously.
+`,
+		CreateContext: resourceOrganizationUserCreate,
+		ReadContext:   resourceOrganizationUserRead,
+		DeleteContext: resourceOrganizationUserDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: schemautil.DefaultResourceTimeouts(),
+
```
Comment (byashimov) [inline internal/sdkprovider/service/organization/organization_user.go:67]: Empty line.

File: internal/sdkprovider/service/organization/organization_user.go [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+func ResourceOrganizationUser() *schema.Resource {
+	return &schema.Resource{
+		Description: `
+The Organization User resource allows the creation and management of an Aiven Organization User.
+
+During the creation of ` + "`aiven_organization_user`" + `resource, an email invitation will be sent
+to a user using ` + "`user_email`" + ` address. If the user accepts an invitation, they will become
+a member of the organization. The deletion of ` + "`aiven_organization_user`" + ` will not only
+delete the invitation if one was sent but not yet accepted by the user, it will also 
+eliminate the member from the organization if one has accepted an invitation previously.
```
[the remark this answers is not in the document]
Comment (Serpentiel) [reply internal/sdkprovider/service/organization/organization_user.go:58]: I never wrote this in first place, it’s copy pasted from `aiven_account_team_member` 😃

File: internal/sdkprovider/service/organization/organization_user.go [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+				return diag.FromErr(err)
+			}
+			if err := d.Set("create_time", invite.CreateTime); err != nil {
+				return diag.FromErr(err)
+			}
+
+			// if a user is in the invitations list, it means invitation was sent but not yet accepted
+			if err := d.Set("accepted", false); err != nil {
+				return diag.FromErr(err)
+			}
```
[the remark this answers is not in the document]
Comment (Serpentiel) [reply internal/sdkprovider/service/organization/organization_user.go:123]: no

File: internal/sdkprovider/service/organization/organization_user.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (Serpentiel) [reply internal/sdkprovider/service/organization/organization_user.go:58]: does not works

**Review by ivan-savciuc — APPROVED** [review]
LGTM

**Pull request merged**